docs/source/performance.rst

```diff
@@ -0,0 +1,174 @@
+Performance Best Practices
+==========================
+
+This is a guide to maximize the performance of Chainer.
```

Comment:
Consider rewording: This is a guide to maximize the performance of Chainer.  ->
This guide explains some tips and advice for maximizing the performance of Chainer.